### ISSUE TITLE
Long-term load (15min avg) larger than number of processors: 13.1 > 1

### ISSUE THREAD

**Issue by Paft** [DesktopECHO/Pi-hole-for-Android#2]
First of all--thanks for the release.

I am new to Pi-hole and don't have a Pi, but do have this old phone. 

I noticed straight after the install that the loads are very high.

Linux Deploy is installed on the system's storage, but the **'Source path'** and **'Installation path'** are on an SD card.

Active Temp: 32.0 °C
**Load: 12.37 12.23 12.27**
Memory usage: 64.4 %

**_Phone Specs:_**
<<< System >>>

Device Model: Motorola Moto G 4G
Device Type: Phone
Manufacturer: motorola
Model: XT1039
Brand: motorola
Board: MSM8226
Device: peregrine
Hardware: qcom
Platform: msm8226
Product: peregrine_retgb
Serial: TA98602CII
Installed RAM: 1 GB LPDDR2
Total Memory: 873 MB
Available Memory: 186 MB
Internal Storage Total Space: 5653 MB
Internal Storage Free Space: 1651 MB
External Storage Total Space: 7493 MB
External Storage Free Space: 5103 MB

<<< CPU >>>

SoC Model: Qualcomm Snapdragon 400 (MSM8926)
Core Architecture: 4x ARM Cortex-A7 @ 1190 MHz
Manufacturing Process: 28 nm
Instruction Set: 32-bit ARMv7
CPU Revision: r0p3
CPU Cores: 4
CPU Clock Range: 300 - 1190 MHz
Core 1 Clock: 1190 MHz
Core 2 Clock: 1190 MHz
Core 3 Clock: 1190 MHz
Core 4 Clock: 1190 MHz
CPU Utilization: 66 %
Scaling Governor: interactive
Supported ABIs: armeabi-v7a, armeabi
Supported 32-bit ABIs: armeabi-v7a, armeabi
AES: Not Supported
NEON: Supported
PMULL: Not Supported
SHA1: Not Supported
SHA2: Not Supported

Full AIDA64 Info:
[AIDA64 for Android v1.85 Report1.txt](https://github.com/DesktopECHO/Pi-hole-for-Android/files/7857168/AIDA64.for.Android.v1.85.Report1.txt)

**Comment by DesktopECHO** [comment]
I saw this on a not-too-old Alcatel One that came with Android Go.  The load average was around 15, all the time, but the phone was cool and Pi-hole was responsive.  Checking online, it appears [some Android devices](https://forum.xda-developers.com/t/q-super-high-load-average.1523693/#post-50537735) just display scary-high load average numbers and that's normal.

**Comment by Paft** [comment]
As I said, I've never used Pi-Hole... so I wouldn't know the difference, but it seems responsive enough. I compared the admin page navigation to random users on Youtube---seems around the same.

Thank you for the very quick reponce and info.

**Comment by DesktopECHO** [comment]
You’re welcome, good luck with it!

**Comment by Paft** [comment]
Sorry to bother you with a mundane question(s)... but is there a way to turn off the screen (to save energy)? I've noticed after x amount of time the WiFi drops when locked. I see a 'Wake Lock' option, but I don't think it helps. I know this is more of an android/Linux Deploy issue, but I thought I'd be cheeky and ask--since this phone will be connected to a charger constantly.

**Comment by DesktopECHO** [comment]
Someone else asked about that today... See https://github.com/DesktopECHO/Pi-hole-for-Android/issues/3
TL;DR: Worst case scenario - Set brightness to its lowest setting, or use the screen for real-time stats.

**Comment by Paft** [comment]
> Someone else asked about that today... See #3 TL;DR: Worst case scenario - Set brightness to its lowest setting, or use the screen for real-time stats.

I flashed from the stock rom to 'lineage-16.0-202112-17-UNOFFICIAL-peregrine.zip' / [ROM][UNOFFICIAL][9.0] LineageOS 16.0 [PEREGRINE]

**Seems to stay connected to the Wi-Fi with the screen off/locked for 7+ hrs so far.**

As for load numbers...they were even high/the same in Developer options>show CPU usage. They're a little lower now (it varies) because of the rom and I don't have GApps installed yet. **9.03  9.73  10.62.**

**Comment by ashishkotnala29** [comment]
I can help you regarding this issue. On a rooted device you can install this: https://play.google.com/store/apps/details?id=eu.thedarken.wl&hl=en&gl=US
Set it to Partial wake lock. Works amazing.
